### PR TITLE
Fix undefined array key "id" in ticket form.

### PR DESCRIPTION
**Pull request by SteadEXE**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #10535

### CODE REVIEW

File: front/ticket.form.php [hunk 1]
```diff
@@ -281,7 +281,7 @@
         Html::header(sprintf(__('%s Kanban'), Ticket::getTypeName(1)), '', "helpdesk", "ticket");
         $track::showKanban(0);
         Html::footer();
-    } else {
+    } else if (isset($_GET['id'])) {
```
Comment (cedric-anne) [inline front/ticket.form.php:284]: There is a `unset($_GET['id']);` on line 263. So, it will never reach this piece of code.

**Review by orthagh — CHANGES_REQUESTED** [review]
with the proposed change, the form doesn't display anymore

I think a better fix should be 

`Ticket::displayFullPageForItem(($_GET["id"] ?? 0), $menus, $_REQUEST);`
at line 286

the change is `($_GET["id"] ?? 0)`

**Comment by SteadEXE** [comment]
It fixes the issue, but it's a weird behavior anyway, there is an `unset($_GET['id']);` on line 263 and a few line later we try to access it `Ticket::displayFullPageForItem($_GET["id"], $menus, $_REQUEST);`.

There is probably a wrong logic there.

**Comment by SteadEXE** [comment]
Yeah, you replied, while I was replying, I saw it too

File: front/ticket.form.php (hunk 1 above)
Comment (SteadEXE) [reply front/ticket.form.php:284]: In fact we are on the else scope of the condition `if (isset($_GET["id"]) && ($_GET["id"] > 0))` line 227. The id can't exist there. I think we can safely remove the else completely.

**Pull request closed without merging**